src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Security/Voter/ManageEntityAccessGrantedVoter.php

```diff
@@ -38,10 +38,15 @@ public function __construct(EntityServiceInterface $entityService)
 
     protected function supports($attribute, $subject)
     {
-        $supportedAttribute = $attribute === self::MANAGE_ENTITY_ACCESS;
-        $subjectFieldsPresent = isset($subject['manageId']) && isset($subject['environment']);
+        if ($attribute !== self::MANAGE_ENTITY_ACCESS) {
```

Comment:
This change had me confused for a minute. You wrote in your commit message this file needed a change, and it did. I only later figured out that the subject parameter was causing you difficulties. What I personally would have done, was input validate the subject parameter. If it is instance of `Request`, then this voter is not supported. 

But I can live with this solution too.

Reply:
I've looked into this and considered using a VO for the subject to be able to validate it much more reliable but I thought that would cost me too much effort for now. So I kept the rest as-is.